src/providers/ad/ad_gpo.c

```diff
@@ -100,7 +100,6 @@
 /* If INI_PARSE_IGNORE_NON_KVP is not defined, use 0 (no effect) */
 #ifndef INI_PARSE_IGNORE_NON_KVP
 #define INI_PARSE_IGNORE_NON_KVP 0
-#warning INI_PARSE_IGNORE_NON_KVP not defined.
```

Comment:
@mzidek-rh does RHEL-6 ship with the newer ding-libs API? If yes, I guess this is OK, otherwise we should keep the warning..